debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/KafkaConnectRecordParser.java

```diff
@@ -78,13 +85,36 @@ public Record parseRecord(Object keyObj, Object valueObj) {
             Struct key = (Struct) ((SourceRecord) valueObj).key();
 
             if (value == null) {
-                // Ideally, we should have config tombstones.on.delete=false. In case that is not set correctly,
+                // Ideally, we should have config tombstones.on.delete=false. In case that is
+                // not set correctly,
                 // we will get those events where value field = null. Skipping those events.
                 LOGGER.warn("Empty value field in event. Assuming tombstone event. Skipping - {}", valueObj);
                 r.op = "unsupported";
                 return r;
+            } else if (value.schema().field("source") == null) {
```

Comment:
We are already handling tombstone events above, right? What sort of events are you trying to avoid here? Is it the transaction begin/end ones?

Reply:
Yep